### DISCUSSION TITLE
What does each symbol mean?

### DISCUSSION THREAD

**Discussion by GoosePeelings** [TEHE-Studios/ExpandedHelicopterEvents#343] [Q&A]
Many of them seem self-explanatory but things like those attached don't tell me much. They are from this one, are they not? Since they're in the same position on the screen like the rest of the helicopter ones. So, what do they mean?

![20240830011839_1](https://github.com/user-attachments/assets/218be161-2f5c-44db-9e83-ce6fd9b6c2eb)
![20240830020138_1](https://github.com/user-attachments/assets/6175cd82-dcbb-426b-bf21-3a0a26b8cd3b)

**Comment by Chuckleberry-Finn** [comment]
These are not from this mod. This could be from the NPC mod 'Bandits' maybe?
